### DISCUSSION TITLE
Add claude-3-5-sonnet-20240620 in the demo

### DISCUSSION THREAD

**Discussion by Oddity-Official** [danny-avila/LibreChat#5152] [General]
### What happened?

I'm trying to use the older claude-3-5-sonnet-20240620 model within the browser demo of LibreChat. It has a variety of models available, however, claude-3-5-sonnet-20240620 is not present. It was there just a couple of months ago. I'm not sure what happened now. The older claude-3-5-sonnet-20240229 is still there, but no 20240620.

### Steps to Reproduce

1. Open up the browser demo https://librechat-librechat.hf.space/
2. Provide Anthropic API key.
3. Open up model drop-down list. 
4. See that there is no claude-3-5-sonnet-20240620 to be seen.
5. ???
6. Despair.

### What browsers are you seeing the problem on?

Firefox

### Screenshots

![image](https://github.com/user-attachments/assets/ac9f67ad-1726-4b0b-a4f7-0786f9b4060e)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by berry-13** [comment]
added `claude-3-5-sonnet-20240620` to the Anthropic endpoint
